### ISSUE TITLE
Account creation?

### ISSUE THREAD

**Issue by alexblack** [thiagolocatelli/android-stripe-connect#2]
This looks really cool - does it support users who don't yet have a stripe account?  I read that the Stripe Connect oauth flow will let users register for an account if they don't have one.

**Comment by thiagolocatelli** [comment]
Yes, you can create an stripe account when you are presented with the
Stripe oauth screen

**Comment by alexblack** [comment]
Awesome.  Is there a quick way for me to try this out? Is there an app in Google Play that uses this?

I downloaded the source, but haven't been able to get it to build in Android Studio yet.

**Comment by thiagolocatelli** [comment]
You can try using Control for Stripe, an app I helped built and it uses
Stripe Connect

You may have to create a Control account, but go for it and let me know
what you think about the app.

**Comment by alexblack** [comment]
Thanks, I tried that out, that looks really good.

The create-account UI for stripe is a bit involved and small, but, I guess
thats on them.

**Comment by alexblack** [comment]
Hey. I'm working on integrating android-stripe-connect now within my app!

I'm trying to use the development shortcut in testing.  Have you tried
that?  I sign in using connect, then instead of filling out the form I
click the 'skip account form' at the top from stripe, then I arrive back at
my app.  I _expected_ to have an access token, but it doesn't seem like
onConnected or onError get called.

thx

Alex
